service/src/main/python/jcarbon/tensorflow/callbacks.py

```diff
@@ -125,6 +97,98 @@ def on_epoch_end(self, epoch, logs=None):
         self.reports[epoch] = self.last_report.to_frame().assign(
             epoch=epoch).set_index('epoch', append=True)
 
+class JCarbonExperimentCallback(JCarbonChunkingCallback):
+    def __init__(
+            self,
+            addr='localhost:8980',
+            period_ms=DEFAULT_PERIOD_MS,
+            signals=DEFAULT_SIGNALS,
+            chunking_period_sec=DEFAULT_PERIOD_SECS):
+        super().__init__(addr, period_ms, signals)
```

Comment:
Why is the chunking period not an argument to the super constructor of the `JCarbonChunkingCallback`? I know there is a default value, but not propagating an argument without a reason smells like a problem.